l5kit/l5kit/dataset/chop_dataset.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+from pathlib import Path
+
+import numpy as np
+from zarr import convenience
+
+from l5kit.data import ChunkedDataset
+from l5kit.data.zarr_utils import zarr_scenes_chunk
+from l5kit.dataset.select_agents import TH_DISTANCE_AV, TH_EXTENT_RATIO, TH_YAW_DEGREE, select_agents
+from l5kit.evaluation import export_zarr_to_csv
+
+MIN_FUTURE_STEPS = 10
+FUTURE_STEPS = 50
+
+
+def create_chopped_dataset(
```

Comment:
I'm a bit confused about the naming here: if I understand it correctly, the chopped zarr (dataset) is actually created by zarr_scenes_chunk(), whereas here we 'just' compute the mask and export the GT csv?

Reply:
no, we also store the chopped dataset here. But I agree that we should change chunk to chop everywhere for clarity.
This function is a high level abstraction that puts together a couple of other utilities.
It is included only to be transparent about how we build the chopped dataset....I'm not really sure this is the right place for it in fact, what do you think @oliver-scheel ?

Reply:
That makes sense. Minor, but in line 58 there is a comment # create chopped dataset and then zarr_scenes_chop is called -> this was my thinking, the comments with the name of this function points to zarr_scenes_chop - but I am fine with the current version.
Regarding where to put it, yeah, this does not actually need to be in L5Kit right? No one uses it, but it is just for showing how we do it. It is probably still nice to keep it as it belongs here - where: I also do not have a better idea. One could think of a folder 'competition', but if this is the only script in there, it does not fit better than here imo.

Reply:
we use it the prediction notebook to simulate what will happen during the competition, but we could have also just shipped the validation set already chopped with the mask and the gt....